.vib/airflow/cypress/cypress/integration/airflow_test_suite.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+import { random } from "./utils";
+
+it("allows user to log in and log out", () => {
+  cy.login();
+  cy.contains("DAGs").should('be.visible');
+  cy.contains("div", "Invalid login").should("not.exist");
+
+  cy.visit("/logout").then(() => {
+    cy.clearCookies();
+    cy.reload();
+    cy.contains("DAGs").should('not.exist');
+  });
+
+});
+
+it("allows triggering execution of a sample DAG", () => {
+  cy.login();
+  cy.fixture("DAGs").then((dags) => {
+    const TRIGGERED_OK_MESSAGE = `Triggered ${dags.triggered.id}, it should start`;
+
+    cy.get("#dag_query")
+      .clear()
+      .type(`${dags.triggered.id}{enter}`);
+    cy.get(`a[href="/tree?dag_id=${dags.triggered.id}"]`).should("be.visible").click();
+    cy.get('a[aria-label="Trigger DAG"]').click();
+    cy.contains("Trigger DAG").click(); // A dropdown appears and clicking again is needed
+    cy.get(".alert-message").should("contain.text", TRIGGERED_OK_MESSAGE);
+  });
+
+  // Verify the DAG appears in the list of active jobs
+  cy.visit("home?status=active");
+  cy.get('span[class="switch"]').should("have.length", 1);
+});
+
+it("checks configuration is not exposed", () => {
+  const SEC_NOTICE =
+    "Your Airflow administrator chose not to expose the configuration, most likely for security reasons";
+
```

Comment:
Could this also be moved to 'fixtures' folder?

Reply:
I don't think it belongs there. `fixtures` are test data, such as users to be created or the like. This is not test data, but a security notice found in the UI (it's like any other selector, but this one is longer and more meaningful).

Reply:
All right!